src/assets/toolkit/scripts/lib/component/slideshow.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+import {u} from 'umbrellajs';
+export class Slideshow {
+  constructor (element, {
+    slideHolder = u('.js-Slideshow-slides'),
```

Comment:
I still would like to see this follow the suggestion [made here](https://github.com/cloudfour/cloudfour.com-patterns/pull/350#discussion_r72332915). We could also refer to [`comment-reply.js`](https://github.com/cloudfour/cloudfour.com-patterns/blob/master/src/assets/toolkit/scripts/lib/component/comment-reply.js) as it does something similar in that these are only string CSS selectors (and not Umbrella objects). They then get wrapped as Umbrella objects [within the `constructor()` method](https://github.com/cloudfour/cloudfour.com-patterns/blob/master/src/assets/toolkit/scripts/lib/component/comment-reply.js#L31).

Let me know if this isn't making sense, though. 😉